File: app/src/main/java/com/esteban/miformacionctma/Actividad.kt
package com.esteban.miformacionctma

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "actividades")
data class Actividad(
    @PrimaryKey(autoGenerate = true)
    val id: Int = 0,
    val titulo: String,
    val descripcion: String,
    val fecha: String,
    val prioridad: String,
    val progreso: Int
)

File: backend/app/models/actividad.py
from sqlalchemy import Column, Integer, String, Text
from app.database import Base


class Actividad(Base):
    __tablename__ = "actividades"

    id = Column(Integer, primary_key=True, index=True, autoincrement=True)
    titulo = Column(String(200), nullable=False)
    descripcion = Column(Text, nullable=True)
    fecha = Column(String(10), nullable=False)
    prioridad = Column(String(10), nullable=False, default="Media")
    progreso = Column(Integer, nullable=False, default=0)